metricbeat/mb/testing/testdata.go

```diff
@@ -242,22 +265,25 @@ func runTest(t *testing.T, file string, module, metricSetName string, config Dat
 			err, module, metricSetName)
 	}
 
+	expectedFile := filepath.Join(config.WritePath, filename+expectedExtension)
```

Comment:
Can you just do the concatenation only if the `WritePath` is not empty? In this, the code will imply that `WritePath` is sth optional.

Reply:
I guess, but by default `writepath` is never empty.

Reply:
Got it, thanks!